src/api/app/models/bs_request.rb

```diff
@@ -73,7 +73,22 @@ class BsRequest < ApplicationRecord
   has_many :bs_request_actions, dependent: :destroy
   has_many :reviews, dependent: :delete_all
   has_many :comments, as: :commentable, dependent: :destroy
-  has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
+  # NOTE: The "where" is forcing the STI to load all possible inheritance
+  # variants. This does not work in development because we do not eager load
+  # classes, while in production it works flawlessly.
```

Comment:
```suggestion
  # variants. This is needed because we in the development environment the elements are not eager loaded by default, while in production it would work flawlessly without it.
```